ecsact/cli/commands/codegen/codegen.cc

```diff
@@ -101,9 +107,44 @@ auto ecsact::cli::codegen(codegen_options options) -> int {
 		}
 
 		for(auto package_id : package_ids) {
-			plugin_fn(package_id, options.output_write_fn);
-		}
+			fs::path output_file_path = ecsact_meta_package_file_path(package_id);
+			if(output_file_path.empty()) {
+				ecsact::cli::report_error( //
+					"Could not find package source file path from ecsact meta "
+					"file package {}\n",
+					output_file_path.string()
```

Comment:
we just checked that this was empty and then are printing it in the error. Doesn't really make sense haha